src/components/Match/matchColumns.jsx

```diff
@@ -1029,6 +1029,53 @@ export const teamfightColumns = [
   },
 ];
 
+const computeAverage = (row, type) => {
+  const t = type === 'obs' ? 'ward_observer' : 'ward_sentry';
+  const maxDuration = items[t].attrib.find(x => x.key === 'lifetime').value;
+  const totalDuration = [];
+  row[`${type}_log`].forEach((ward) => {
+    const findTime = row[`${type}_left_log`].find(x => x.ehandle === ward.ehandle);
+    const leftTime = (findTime && findTime.time) || row.duration;
+    const duration = Math.min(Math.max(leftTime - ward.time, 0), maxDuration);
+    totalDuration.push(duration);
+  });
+  let sum = 0;
+  for (let i = 0; i < totalDuration.length; i += 1) {
+    sum += totalDuration[i];
+  }
+  const avg = sum / totalDuration.length;
+
+  return avg;
+};
+
+const obsAvgColumn = {
+  center: true,
+  displayName: (
+    <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+      <img height="15" src={`${process.env.REACT_APP_API_HOST}/apps/dota2/images/items/ward_observer_lg.png`} alt="" />
+      &nbsp;{strings.th_duration_shorthand}
+    </div>
+  ),
+  tooltip: strings.tooltip_duration_observer,
+  sortFn: true,
+  displayFn: row => formatSeconds(computeAverage(row, 'obs')) || '-',
+  relativeBars: true,
```

Comment:
Looks like the bars aren't working

Reply:
Ops I totally forgot about the bars. I'll try to do something about them.

Reply:
I think they use the displayFn value so if it's a string it behaves strangely.  We handled this for time dead though so there's a workaround